fundamentals/examples.tex

```diff
@@ -260,12 +290,11 @@ \subsection{Simulations}
 \label{fig:puseries1}
 \end{figure}
 
-The open, modified open, and closed scenarios can easily be modified to have
-several reactors with staggered refueling times. The plutonium buildup
-for these three multi-reactor variations is shown in Figure
-\ref{fig:puseriesn}. As the number of
-reactors increases, the behavior of the system approaches a more steady
-average reminiscent of continuous material flow models.
+Figure \ref{fig:puseriesn} shows plutonium buildup for the 10-reactor
+simulations of the open, modified open, and closed scenarios.  As the number
+of reactors (with staggered refueling) increases, the behavior of the system
+approaches a more steady average reminiscent of continuous material flow
+models.
```

Comment:
These graphs demonstrate the right things, I think. My final comment is just that this section might do well to have a wrap up after the final figure. A couple of sentences like "These demonstrations have show the fundamental capabilities of Cyclus as well as the breadth of scenario types that are possible with the Cyclus ecosystem. These examples further show the flexibility provided by the material routing logic within cyclus and the importance of discrete facility and discrete material modeling in fuel cycle simulation".... something like that.